packages/core-snapshots/lib/db/index.js

```diff
@@ -71,7 +76,7 @@ module.exports = class Database {
 
     if (!startBlock || !endBlock) {
       logger.error('Wrong input height parameters for building export queries. Blocks at height not found in db.')
-      process.exit(1)
+      throw new Error('Wrong input height parameters for building export queries. Blocks at height not found in db.')
```

Comment:
This should be a `process.exit(1)` to 100% make sure the application actually exits or set `process. exitCode = 1` before you throw the error.